README.md

```diff
@@ -59,4 +155,8 @@ Once this is all set up and you aren't getting any errors you can run our test s
 
 - `$ bundle exec rspec`
 
-If you would rather enjoy the application in its finished form without messing with the command line, we are hosted on Heroku [here]().
+If you would rather enjoy the application in its finished form without messing with the command line, we are hosted on Heroku [here](https://obituary-hub.herokuapp.com/).
+
+## Project Board
```

Comment:
This implies we will be cleaning up our project board before deployment so that others can join and understand exactly what needs to be done. Not sure if we will have time, but I guess that's the goal.